### PR TITLE
A Skylark Driver

### PR DESCRIPTION
**Pull request by mfine**
An initial skylark driver. Takes a `device-uid` to pull observations for. 

/cc @mookerji @fnoble

### CODE REVIEW

**Comment by mookerji** [comment]
That was fast.

File: python/sbp/client/drivers/skylark_driver.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  Parameters
+  ----------
+  device_uid : uid
+    Device unique id.
+
+  """
+  def __init__(self, device_uid):
+    url = 'http://skylark.swiftnav.com'
+    headers = {'Device-Uid': device_uid, 'Accept': 'application/sbp'}
```
Comment (mookerji) [inline python/sbp/client/drivers/skylark_driver.py]: POST? Error handling and retries?

Comment (mfine) [inline python/sbp/client/drivers/skylark_driver.py]: This is the client, receive side. Shoudl we setup the server, send side in the same driver?

Comment (mookerji) [inline python/sbp/client/drivers/skylark_driver.py]: This is vaguely related to the refactoring question. We still need a client POST to the endpoint. We could expose a file handle interface for HTTP and stream through that.

**Pull request merged**